themes/openemr/layouts/partials/navigation.html

```diff
@@ -18,6 +18,7 @@
           <a href="" class="nav-link dropdown-toggle" id="modulesDropdownMenuLink" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Modules</a>
           <div class="dropdown-menu" aria-labelledby="modulesDropdownMenuLink">
             <div class="arrow-up"></div>
+            <a class="dropdown-item" href="{{ "https://community.open-emr.org/t/v2-2-0-fax-sms-module-released/16157" }}">Fax/SMS</a>
```

Comment:
entries that go here will require entry on the OpenEMR_Modules wiki page (would make a category for this like is done for the telehealth category). That entry will require a quick description, production version(s) of OpenEMR that it works in and then a link (or instruction set in the entry itself) to a clear instruction set for setup, configure, and use.

Reply:
Would prefer to just create this in hugo instead of pointing to the wiki

Reply:
@robertdown , your website changes look great! But am keeping the bar very high for modules that get listed. For the fax/sms module to get in there would need the following:
1. Clear instruction set(s) for setup, configuration, and use
2. The code itself needs a code review (ie. mainly for a security audit)